testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksDelete.java

```diff
@@ -53,4 +58,50 @@ public void testDeleteOnAppendOnlyTableFails()
                 .containsOnly(row(1, 11), row(2, 12));
         onTrino().executeQuery("DROP TABLE " + tableName);
     }
+
+    // Databricks 12.1 added support for deletion vectors
+    // TODO: Add DELTA_LAKE_OSS group once after upgrading OSS Delta Lake image to 2.3.0
```

Comment:
This is already available after merging https://github.com/trinodb/trino/pull/16957